velox/exec/tests/TableWriteTest.cpp

```diff
@@ -917,6 +917,61 @@ class TableWriteTest : public HiveConnectorTestBase {
   core::PlanNodeId tableWriteNodeId_;
 };
 
+class BasicTableWriteTest : public HiveConnectorTestBase {};
+
+TEST_F(BasicTableWriteTest, roundTrip) {
```

Comment:
Why need this test? thanks!

Reply:
Just to have a simple example of creating a plan with a table writer. Useful to tweak when reproducing production issues.